index.bs

```diff
@@ -115,8 +115,100 @@ The <dfn for="a" element-attr>attributionreportto</dfn> attribute optionally dec
 The <dfn for="a" element-attr>attributionexpiry</dfn> attribute optionally defines the amount
 of time in milliseconds the attribution source should be considered for reporting.
 
-Issue: Need monkey patches passing attribution source in navigation, and a mechanism
-for validating the resulting document matches the attributiondestination.
+<h3 id="monkeypatch-navigation">Navigation</h3>
+
+The user agent must verify that an [=attribution source=] associated with a navigation
+results in a top-level navigation whose final url is [=same site=]
+[=attribution source/attribution destination=].
+
+<h4 id="monkeypatch-navigation-params">Navigation Params</h4>
+
+A <a spec="HTML">navigation params</a> struct has an item:
+
+: <dfn lt="navigation params attribution source" id="navigation-params-attribution-source">attribution source</dfn>
+:: an optional [=attribution source=] declared when initiating a navigation
```

Comment:
It looks like the other items are "null or X" instead of "an optional X", so try to follow that here.